fluss-client/src/main/java/org/apache/fluss/client/admin/Admin.java

```diff
@@ -492,4 +503,90 @@ ListOffsetsResult listOffsets(
      * @return A CompletableFuture indicating completion of the operation.
      */
     CompletableFuture<Void> alterClusterConfigs(Collection<AlterConfig> configs);
+
+    /**
+     * Add server tag to the specified tabletServers, one tabletServer can only have one serverTag.
+     *
+     * <p>If one tabletServer failed adding tag, none of the tags will take effect.
+     *
+     * <ul>
+     *   <li>{@link AuthorizationException} If the authenticated user doesn't have cluster
+     *       permissions.
+     *   <li>{@link ServerNotExistException} If the tabletServer in {@code tabletServers} does not
+     *       exist.
+     *   <li>{@link ServerTagAlreadyExistException} If the server tag already exists for any one of
```

Comment:
The method `addServerTag` should be **idempotent**: if some servers already have the given tag, it should succeed without error. Only when a server already has a **different** server tag (i.e., not equal to the one being set) should it throw `ServerTagAlreadyExistException`. Could you update the javadoc here?